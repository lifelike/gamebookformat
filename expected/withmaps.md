# Example Gamebook With Maps #
Pelle Nilsson

Turn to 1 to begin.
<a name="section10">

## 10 ##
 This should have number 10, as specified in first map file. The end, at [99](#section99)
, should be at 99. 
<a name="section99">

## 99 ##
 Number 99, as specified in second map file.

