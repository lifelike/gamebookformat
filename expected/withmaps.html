<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Example Gamebook With Maps</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'withmaps';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section10"
     >
<div class="sectionnumber" id="para10">10</div>
<div class="sectiontext">
 This should have number 10, as specified in first map file. The end, at <a class="sectionref enabledlink" data-ref="99"
href="#section99">99</a>, should be at 99. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(10, document.getElementById('section10'));
}
</script>
<div class="section" id="section99"
     >
<div class="sectionnumber" id="para99">99</div>
<div class="sectiontext">
 Number 99, as specified in second map file.
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(99, document.getElementById('section99'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
